stdlib/Dates/src/deprecated.jl

```diff
@@ -68,3 +67,21 @@ end
 
 @deprecate argerror(msg::String) ArgumentError(msg) false
 @deprecate argerror() nothing false
+
+import Base: ceil, round, floor
+
+Base.@deprecate(round(dt::TimeType,  p::Period, r::RoundingMode{:NearestTiesUp}), round(p, dt, r))
```

Comment:
`@deprecate` is exported form base and also (redundantly) used at the top of this file, so there is no need to qualify it here.

Reply:
I prefer the macro call syntax `@m e1 e2` over `@m(e1, e2)` when possible (see usages of `@deprecate` elsewhere in this file)

Reply:
Changed it to `@m e1 e2`